apps/web/src/app/user/my-claims/[id]/page.tsx

```diff
@@ -0,0 +1,63 @@
+"use client";
+
+import { useParams } from "next/navigation";
+import React, { useEffect, useState } from "react";
+import ProductStatusDetails from "~/app/_components/features/ProductStatusDetails";
+import { ProfileOptionLayout } from "~/app/_components/features/ProfileOptionLayout";
+
+type SaleDetails = {
+	productName: string;
+	status: string;
+	roast: string;
+	type: string;
+	quantity: string;
+	delivery: string;
+	totalPrice: string;
+	address?: string;
+};
+
+export default function MySaleDetails() {
```

Comment:
I copy this class from [my sales](https://github.com/Vagabonds-Labs/cofiblocks/blob/main/apps/web/src/app/user/my-sales/%5Bid%5D/page.tsx) just changing the status to delivered... not sure if any other change is needed here. I noticed that in the Figma there is an extra option for "Talk with the seller", not sure how to mock that, just show the option and do nothing else?